src/test/java/uk/ac/ebi/ena/webin/cli/context/genome/GenomeManifestReaderFileCountTest.java

```diff
@@ -103,11 +102,24 @@ public void testPrimaryMetagenomeFileCount() {
 
   @Test
   public void testClinicalIsolateAssemblyFileCount() {
+    final List<GenomeManifest.FileType> fileTypeList = getFileTypes();
+
     new ManifestReaderFileCountTester<>(
-            GenomeManifestReader.class, GenomeManifest.FileType.values())
+            GenomeManifestReader.class, fileTypeList.toArray(new GenomeManifest.FileType[0]))
         // Supported file groups
         .files(GenomeManifest.FileType.FASTA)
         .field("ASSEMBLY_TYPE", "clinical isolate assembly")
         .test();
   }
+
+  private static List<GenomeManifest.FileType> getFileTypes() {
+    /* We are keeping webin-cli-validator and sequencetools unchanged,
+    and hence we get AGP file type from GenomeManifest.FileType,
+    we exclude it upfront */
```

Comment:
This comment is enough here, can be removed from other places.

Reply:
Done.